src/Sylius/Component/Core/Currency/ChannelAwareCurrencyProvider.php

```diff
@@ -48,4 +49,18 @@ public function getAvailableCurrencies()
             return $currency->isEnabled();
         });
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getBaseCurrency()
+    {
+        $currentChannel =  $this->channelContext->getChannel();
+
+        foreach ($currentChannel->getCurrencies() as $currency) {
```

Comment:
ah ok, maybe I'm confusing this one as I see the method on the `CurrencyProvider` below. 

What is this for?